lib/vc.js

```diff
@@ -62,93 +66,237 @@ module.exports = {
  * @param {ProofPurpose} [options.purpose]
  * @param {string} [options.controller]
  * @param {string} [options.domain]
- * @param {string} [options.challenge]
  *
  * @param {string|Date} [options.issuanceDate]
  *
  * Other optional params passed to `sign()`:
  * @param {function} [options.documentLoader]
  * @param {Object} [options.expansionMap]
  * @param {boolean} [options.compactProof]
+ *
+ * @throws {Error}
+ *
+ * @returns {Promise<object>}
  */
 async function issue(options = {}) {
   const documentLoader = options.documentLoader || defaultDocumentLoader;
 
-  const {controller, domain, challenge} = options;
+  const {controller, domain} = options;
   const purpose = options.purpose || new CredentialIssuancePurpose({
     controller,
-    domain,
-    challenge
+    domain
   });
 
   // run common credential checks
-  await _checkCredential({credential: options.credential});
+  const {credential} = options;
+  if(!credential) {
+    throw new TypeError('"credential" property is required for issuing.')
+  }
+  _checkCredential(credential);
 
-  return jsigs.sign(options.credential, {purpose, documentLoader, ...options});
+  return jsigs.sign(credential, {purpose, documentLoader, ...options});
 };
 
 /**
- * Verify a credential.
+ * Verifies a verifiable credential or verifiable presentation.
+ *
+ * @param {object} [options={}]
+ *
+ * Required, one of:
+ * @param {object} [options.credential] - Verifiable credential
+ * @param {object} [options.presentation] - Verifiable Presentation
```

Comment:
General comment that the capitalization, or not, of these should be consistent throughout the code.  Maybe match what the vc-data-model spec is doing?  I think they mostly use lower case in text.